matador/compute/compute.py

```diff
@@ -2049,8 +2049,23 @@ def _setup_compute_dir(self, seed, compute_dir, custom_params=False, generic=Fal
         # if a checkfile exists, copy it to the new dir
         # so that it can be restarted from
         if os.path.isfile(seed + ".check"):
+            LOG.info("Copying .check into compute_dir")
             shutil.copy2(seed + ".check", compute_dir)
 
+        # if a magres .mag checkfile exists, copy it to the new dir
+        # so that it can be restarted from for a magres task
+        mag_files = glob.glob(seed + ".*.mag")
```

Comment:
Whats the pattern of the filenames here?

Whilst its not a big issue for these kind of workflows as you're probably doing at most 100 calcs at once, doing a wildcard glob in the top directory can be expensive on some filesystems (like the infamous pre-launch of CSD3, where I was using 50% of my CPU time running the equivalent of `ls *` in a directory with 100000 AIRSS trials 🙃).

Assuming its `<seed>.0001.mag` or something, for each process? In which case we could check for 0001, then 0002 etc. which will be faster than the glob.

Reply:
Yes this is correct so I have changed this to:
```mag_files = [
            f"{seed}.{num:04}.mag" for num in range(1, self.ncores * self.nnodes)
        ]  
        if os.path.isfile("%s.0001.mag" % (seed)):
```

Reply:
Let me know if this is sufficient/sensible